Realm/RLMObjectBase.mm

```diff
@@ -654,6 +663,7 @@ - (void)invalidate {
 - (void)addNotificationBlock:(RLMObjectNotificationCallback)block
          threadSafeReference:(RLMThreadSafeReference *)tsr
                       config:(RLMRealmConfiguration *)config
+                    keyPaths:(KeyPathArray)keyPaths
```

Comment:
It would make sense. But there are several places in the cocoa code base where the pattern of adding an `s` to indicate multiple already exists, so 1) I try to match that 2) and it's four characters shorter :)